dv/fcov/dv_fcov.core

```diff
@@ -0,0 +1,17 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:dv_fcov"
+description: "DV FCOV utilities"
+
+filesets:
+  files_fcov:
+    files:
+      - dv_fcov.svh: {is_include_file: true}
```

Comment:
Generally suffix files that only contain macros: example: `dv_fcov_macros.svh`.

Reply:
I've renamed it to `dv_fcov_macros.svh`